core/tests/protractor_desktop/preferences.js

```diff
@@ -112,12 +112,12 @@ describe('Preferences', function() {
     await waitFor.pageToFullyLoad();
 
     await preferencesPage.expectPreferredAudioLanguageToBe('Hindi');
-    await preferencesPage.selectPreferredAudioLanguage('Arabic');
-    await preferencesPage.expectPreferredAudioLanguageToBe('Arabic');
+    await preferencesPage.selectPreferredAudioLanguage('Hungarian');
+    await preferencesPage.expectPreferredAudioLanguageToBe('Hungarian');
     await browser.refresh();
     await waitFor.pageToFullyLoad();
 
-    await preferencesPage.expectPreferredAudioLanguageToBe('Arabic');
+    await preferencesPage.expectPreferredAudioLanguageToBe('Hungarian');
```

Comment:
Why change this?

Reply:
protractor had trouble scrolling `mat-select`. So, I tried to use a select option, which didn't require scrolling.

Reply:
If this is an issue with protractor and mat-select in general, can you please create an issue and assign it to the Automated QA team to investigate? We will need to find a way to fix this in case a test we write in the future uses mat-select and actually expects scrolling to be done.

Reply:
sure

Reply:
done. Issue #13324